cmd/limactl/edit.go

```diff
@@ -53,17 +58,37 @@ func editAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	hdr := fmt.Sprintf("# Please edit the following configuration for Lima instance %q\n", instName)
-	hdr += "# and an empty file will abort the edit.\n"
-	hdr += "\n"
-	hdr += editutil.GenerateEditorWarningHeader()
-	yBytes, err := editutil.OpenEditor(instName, yContent, hdr)
+	tty, err := cmd.Flags().GetBool("tty")
 	if err != nil {
 		return err
 	}
-	if len(yBytes) == 0 {
-		logrus.Info("Aborting, as requested by saving the file with empty content")
-		return nil
+	yq, err := cmd.Flags().GetString("set")
+	if err != nil {
+		return err
+	}
+	var yBytes []byte
+	if yq != "" {
+		logrus.Warn("`--set` is experimental")
```

Comment:
https://github.com/lima-vm/lima/blob/master/docs/experimental.md has to be updated too